src/pestuary.py

```diff
@@ -182,8 +182,8 @@ def content_list():
 
 
 # creates a new pin
-def pin_create(cid, name):
```

Comment:
what if we do this:
def pin_create(cid, name):
  return pinningApi.pinning_pins_post({"cid":cid,"name":name})